imap_processing/mag/l0/mag_l0_data.py

```diff
@@ -90,19 +94,24 @@ class MagL0:
     PRI_FNTM: int
     SEC_COARSETM: int
     SEC_FNTM: int
-    VECTORS: bytearray
+    VECTORS: np.ndarray | str
```

Comment:
Can you add in the doc string why this would be either array or string? This seems like unusual case.

Reply:
Sure!